include/sys/vm_amap.h

```diff
@@ -1,45 +1,97 @@
 #ifndef _SYS_VM_AMAP_H_
 #define _SYS_VM_AMAP_H_
 
+#include <sys/mutex.h>
```

Comment:
Is that include really necessary here?

Reply:
I have removed these includes, because they were unnecessary, but I had to add other ones (included in `sys/mutex.h` to correctly build mimiker).